server/models/emailAutomation.js

```diff
@@ -32,16 +20,11 @@ export default (sequelize, DataTypes) => {
         type: DataTypes.STRING,
         allowNull: false,
       },
-    },
-  ));
+    }),
+  );
 
   EmailAutomation.associate = (models) => {
-    EmailAutomation.belongsTo(models.Automation, {
-      foreignKey: {
-        name: 'automationId',
-        allowNull: false,
-      },
-    });
+    EmailAutomation.belongsTo(models.Automation, relationships);
```

Comment:
The `relationships` used here does not contain the constraint of `allowNull: false`, how about it?

Reply:
Why the `allowNull`? I don't think such property has any relevance when defining relationships. I stand to be corrected

Reply:
Yh. You are right, the foreign key constraints should handle it.